#ifndef SRC_CPP_PROTO_MESSAGES_H_
#define SRC_CPP_PROTO_MESSAGES_H_

#include "proto/common.pb.h"
#include "proto/stats.pb.h"
#include "proto/topology.pb.h"
#include "proto/execution_state.pb.h"
#include "proto/physical_plan.pb.h"
#include "proto/tuple.pb.h"
#include "proto/tmaster.pb.h"
#include "proto/scheduler.pb.h"
#include "proto/stmgr.pb.h"
#include "proto/metrics.pb.h"

#endif  // SRC_CPP_PROTO_MESSAGES_H_
